test/test_generate_counterfactuals.py

```diff
@@ -0,0 +1,84 @@
+import os
+import sys
+from test.constants import DATASET_PATHS
+
+import joblib
+import pytest
+
+from training.creating_dataset import load_and_preprocess_data
+
+
+# sys.path.insert(0, os.sep.join(os.path.normpath(__file__).split(os.sep)[:-2]).join("serving/backend"))
+sys.path.append("../..")
```

Comment:
Czy to jest potrzebne przy tym fixie w `pyproject.toml`? Plus można usunąć komentarz, jak już jest niepotrzebny